CHANGELOG.md

```diff
@@ -15,8 +15,8 @@ These changes are available on the `master` branch, but have not yet been releas
 - Added new events `on_bridge_command`, `on_bridge_command_completion`, and
   `on_bridge_command_error`.
   ([#1916](https://github.com/Pycord-Development/pycord/pull/1916))
-- Added the `@client.once()` decorator, which serves as a one-time event listener.
-  ([#1940](https://github.com/Pycord-Development/pycord/pull/1940))
+- Added support for one-time event listeners in `@client.listen()`.
```

Comment:
Please split to removed and added

Reply:
For unreleased (stable) changes too? That would be misleading

Reply:
You can't just override a while change like that
Even if it wasn't released yet, users were able to install it via git@master

Reply:
Alright cool. On it